tools/build-toolchain.sh

```diff
@@ -60,7 +60,7 @@ download () {
 }
 
 # Compilation on macOS via homebrew
-if [[ $OSTYPE == 'darwin'* ]]; then
+if [[ $OSTYPE == *'darwin'* ]]; then
```

Comment:
Why is this required?

Reply:
I cannot remember.
I am tempted to remove the native macos builds altogether (for the moment).